comp/api/api/utils/stream/stream.go

```diff
@@ -93,3 +96,23 @@ func GetStreamFunc(messageReceiverFunc func() MessageReceiver, streamType, agent
 		}
 	}
 }
+
+// OpenFileForWriting opens a file for writing
+func OpenFileForWriting(filePath string) (*os.File, *bufio.Writer, error) {
+	log.Infof("opening file %s for writing", filePath)
+	f, err := os.OpenFile(filePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
Would it make sense to truncate the file here or clean the streaming file in some other way? Otherwise subsequent flares on the same host would keep getting bigger and bigger, possibly including outdated information.